lighthouse/classes/services/mongo.py

```diff
@@ -0,0 +1,28 @@
+from flask import current_app as app
+from lighthouse.constants.fields import FIELD_BARCODE, FIELD_LH_SOURCE_PLATE_UUID, FIELD_LH_SAMPLE_UUID
+
+
+class ServiceMongoMixin:
+    def get_samples_from_mongo(self, uuids):
```

Comment:
I would rename this to `get_samples`.

Reply:
I think only with get_samples we lose where are the samples coming from. I'm thinking that we will see this function written in another place and from there by reading the code we don't know what samples are referring (Mongo, Dart, Warehouse, ....)